web-frontend/src/i18n/en_US.json

```diff
@@ -18,11 +18,14 @@
         "description": "Now, define your password. To maximize security, a pop up will open directly on your device via MassaStation."
       },
       "step3": {
-        "title": "Backup"
+        "title": "Backup",
+        "description": "Security is very important, we recommend backing up your account using a key pair or a .yaml file. Would you like to do it now or later?"
       },
       "buttons": {
         "define_password": "Define a password",
-        "next": "Next"
+        "next": "Next",
+        "backup": "Back up",
```

Comment:
```suggestion
        "backup": "Backup",
```

Reply:
One the design, it's "back up".
So what would I do ?

Reply:
Oh! Check with him then!

Reply:
> Backup is a noun spelled as a single word, while back up is a verb spelled as 2 words

Reply:
here either you skip, or you back up (2 verbs)